apps/web/components/ui/PoweredByCal.tsx

```diff
@@ -12,22 +12,22 @@ const PoweredByCal = () => {
       <Link href={POWERED_BY_URL}>
         <a target="_blank" className="text-bookinglight opacity-50 hover:opacity-100 dark:text-white">
           {t("powered_by")}{" "}
-          {
-            // eslint-disable-next-line @next/next/no-img-element
-            <img
-              className="relative -mt-px inline h-[10px] w-auto dark:hidden"
-              src="/cal-logo-word.svg"
-              alt="Cal.com Logo"
-            />
-          }
-          {
-            // eslint-disable-next-line @next/next/no-img-element
-            <img
-              className="relativ -mt-px hidden h-[10px] w-auto dark:inline"
-              src="/cal-logo-word-dark.svg"
-              alt="Cal.com Logo"
-            />
-          }
+          {APP_NAME === "Cal.com" ? (
+            <>
+              <img
+                className="relative -mt-px inline h-[10px] w-auto dark:hidden"
+                src="/cal-logo-word.svg"
+                alt="Cal.com Logo"
+              />
+              <img
+                className="relativ -mt-px hidden h-[10px] w-auto dark:inline"
+                src="/cal-logo-word-dark.svg"
+                alt="Cal.com Logo"
+              />
+            </>
+          ) : (
+            <span className="font-semibold">{APP_NAME}</span>
```

Comment:
for now, we will keep powered by Cal.com

Reply:
is that an /ee/ feature too?

Reply:
yes

Reply:
is there a chance to get a licence for white labeling in future?

Reply:
@ddm231 it will be an /ee/ license required: https://cal.com/platform